src/controllers/__tests__/item.test.js

```diff
@@ -87,13 +87,12 @@ const noDescriptionItem = {
 describe('Items Endpoints', () => {
   beforeAll(async () => {
     connectToMongo();
-    await clearDatabase();
   });
 
   afterAll(async () => {
     await clearDatabase();
     await closeDatabase();
```

Comment:
It makes more sense to clear database beforeAll instead of afterAll. 
I am still getting an error (ns not found) when I do this way and Jest can't exit after tests are completed.